packages/block-editor/src/components/iframe/index.js

```diff
@@ -378,10 +378,9 @@ function Iframe( {
 			<iframe
 				{ ...props }
 				style={ {
-					border: 0,
 					...props.style,
 					height: props.style?.height,
-					transition: 'all .3s',
+					transition: 'all 0.5s cubic-bezier(0.65, 0, 0.45, 1)', // Maps to the .block-editor-iframe__html transition.
```

Comment:
Why don't we move this to the `iframe/content.scss` file to make it easeir? I don't think we need this defined in the JS anyways.

Reply:
`.edit-site-visual-editor__editor-canvas` already is used in scss [elsewhere](https://github.com/WordPress/gutenberg/blob/1e4d4272cd5435060fef3287a84a3537af0e0473/packages/edit-site/src/components/block-editor/style.scss#L4). 🤷‍♂️

Reply:
Yeah, I think we should move it there. I don't see why it should be defined in the JS over CSS.

Reply:
I agree with defining it in CSS instead of JS but not in `Iframe`’s styles because as long as it’s innate to the component it exacerbates #50449 by slowing down how quickly the pattern preview reaches its final height. It’s one thing I've been meaning to make a PR for. It can be done separately of course. `packages/block-editor/src/components/block-canvas/style.scss` seems a decent place to put it.